drivers/azure/azureutil/timeouts.go

```diff
@@ -0,0 +1,25 @@
+package azureutil
```

Comment:
The timeouts should be moved to where they are needed, one is even exported as it's used in a different package. It also looks like some of these are not used. If we don't need them they can be deleted.

Reply:
And can these be set somehow? If they can't be they should be changed to a const.

Reply:
Yeah that makes sense. Moving them to their respective files in the latest commit.

They shouldn't be set so I'll change them to a const too.